Services/CRUD/CrudTransformer.php

```diff
@@ -77,10 +80,64 @@ public function __construct(
     }
 
     /**
-     * Array to collection transform
+     * Array to object transformation
      *
-     * @param array  $data  Data
-     * @param string $class Given collection class
+     * @param array  $data  Object fields
+     * @param string $class Class name of object to create
+     * @return mixed
+     * @throws ConstraintViolation
+     */
+    public function arrayToObject(array $data, $class)
+    {
+        $this->initializeClassMetadata($class);
+        $object = new $class();
+        if (is_array($data)) {
```

Comment:
I don't think that we need is_array check, if we cast to array in method definition